CMakeLists.txt

```diff
@@ -11,15 +9,12 @@ option(BUILD_SHARED_LIBS "Build shared libraries by default" YES)
 include(CTest)
 include(SwiftSupport)
 
-if(CMAKE_VERSION VERSION_LESS 3.16 AND CMAKE_SYSTEM_NAME STREQUAL Windows)
-  set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
-  set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
-else()
-  set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
-  set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
-endif()
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules)
```

Comment:
Isn't this too late now @compnerd? `SwiftSupport` is within `cmake/modules` and is included before this now. CC @xedin

Reply:
Hmm, that is a good point 🤔.  How did CI get away with this?

Reply:
Ah, looking at logs CI is using SPM, not CMake

Reply:
Oh lovely :(